app/controllers/pageflow/application_controller.rb

```diff
@@ -21,8 +21,14 @@ class ApplicationController < ActionController::Base
 
     rescue_from ActiveRecord::RecordNotFound do
       respond_to do |format|
-        format.html { render :file => Rails.root.join('public/404.html'), :status => :not_found }
-        format.any(:json, :css) { head :not_found }
+        format.html do
+          begin
+            render file: Rails.public_path.join('pageflow', 'error_pages', '404.html'), status: :not_found
```

Comment:
I think it's worth it because the `head :not_found` in ApplicationController _is_ part of our code. I added 527f703 to test this. It might not look pretty but it is effective. I added the safeguard since the test could fail for other reasons and leave the dummy app in an unclean state.

An alternative could be to not bother with file-level stuff (which technically doesn't belong in a controller test anyway) but stub out the call to `render` and raise `ActionView::MissingTemplate` from the test. But then you're testing Rails framework detail which might change at some point...

Not sure what we should prefer here. But I do think a test has value.